src/ContentBundle/Block/Service/ColumnBlockService.php

```diff
@@ -120,13 +120,37 @@ public function buildManageForm(FormBuilderInterface $builder, array $options)
                     'required' => false,
                     'expanded' => true,
                     'multiple' => true,
-                    'attr' => ['help_text' => 'help.html_styles','tag' => 'styles'],
+
+                    'attr' => ['help_text' => 'help.list_display_size', 'class' => 'btn-group btn-group-styling', 'data-toggle' => 'buttons'],
+                    'label_attr' => ['class' => 'btn'],
                 ])
+               ->add('column_count', ChoiceType::class, [
```

Comment:
@WoutGroenendijk , This `column_count` is already present in the current situation. Could you merge your changes with the existing one, so there's only one `column_count` field?